src/els_parser.erl

```diff
@@ -47,20 +47,22 @@ parse_form(IoDevice, Location, Options) ->
   {'ok', erl_syntax:forms() | none, integer()}
   | {'eof', integer()}
   | {'error', any(), integer()}.
-parse_form(IoDevice, Location0, Parser, _Options) ->
-  case io:scan_erl_form(IoDevice, "", Location0) of
-    {ok, Tokens, Location1} ->
+parse_form(IoDevice, StartLocation, Parser, _Options) ->
+  case io:scan_erl_form(IoDevice, "", StartLocation) of
+    {ok, Tokens, EndLocation} ->
       try {ok, Parser(Tokens, undefined)} of
         {ok, F} ->
-          POIs = [find_attribute_pois(F, Tokens), points_of_interest(F)],
-          {ok, POIs, Location1}
+          POIs = [ find_attribute_pois(F, Tokens)
+                 , points_of_interest(F, EndLocation)
+                 ],
+          {ok, POIs, EndLocation}
       catch
         _:_ ->
-          {ok, find_attribute_tokens(Tokens), Location1}
+          {ok, find_attribute_tokens(Tokens), EndLocation}
```

Comment:
I could have also passed the `StartLocation`, but I decided to go for the minimum possible change here.